src/java/htsjdk/samtools/SAMSequenceDictionary.java

```diff
@@ -163,8 +176,124 @@ public boolean equals(Object o) {
         return true;
     }
 
+    /**
```

Comment:
Probably a good idea to update the comment in ```setSequences``` to say that aliases will not be overwritten.